sisl/viz/plotly/input_fields/dropdown.py

```diff
@@ -155,9 +155,9 @@ class SpinSelect(DropdownInput):
     }
 
     _options = {
-        Spin.UNPOLARIZED: [],
+        Spin.UNPOLARIZED: [{"label": "Total", "value": 0}],
         Spin.POLARIZED: [{"label": "↑", "value": 0}, {"label": "↓", "value": 1}],
-        Spin.NONCOLINEAR: [{"label": val, "value": val} for val in ("x", "y", "z")],
+        Spin.NONCOLINEAR: [{"label": val, "value": val} for val in ("sum", "x", "y", "z")],
         Spin.SPINORBIT: []
```

Comment:
spin-orbit is the same as noncolinear

Reply:
But what about here? why the `Spin.SPINORBIT` box? I guess this should have the same as noncolinear?

Reply:
Yes, I will add it, but for now it will have no effect on anything I believe.